packages/solutions/app-tools/src/analyze/constants.ts

```diff
@@ -38,8 +38,12 @@ export const FILE_SYSTEM_ROUTES_COMPONENTS_DIR = 'internal_components';
 export const NESTED_ROUTE = {
   LAYOUT_FILE: 'layout',
   LAYOUT_LOADER_FILE: 'layout.loader',
+  LAYOUT_DATA_FILE: 'layout.data',
+  LAYOUT_CLIENT_LOADER: 'layout.data.client',
```

Comment:
Why not named it as `layout.loader.client` ?

Reply:
Because we will support action later, there will be both loader and action in this file.